pytest.ini

```diff
@@ -2,3 +2,5 @@
 testpaths = kobo kpi hub
 env =
     DJANGO_SETTINGS_MODULE=kobo.settings.testing
+filterwarnings =
+    ignore::django.utils.deprecation.RemovedInDjango20Warning
```

Comment:
This is fine, but let's take all the warnings it generates and put them in a new GitHub issue